orttraining/orttraining/python/training/checkpoint.py

```diff
@@ -108,6 +109,196 @@ def experimental_load_checkpoint(ort_trainer, checkpoint_dir, checkpoint_prefix=
     else:
         return _load_single_checkpoint(ort_trainer, checkpoint_dir, checkpoint_prefix, is_partitioned, strict)
 
+def _order_paths(paths):
+    """Reorders the given paths in ascending order of rank and return the ordered list"""
+
+    trainer_options_path_tuples = []
+
+    for path in paths:
+        trainer_options_path_tuples.append((_checkpoint_storage.load(path,
+            key=_utils.state_dict_trainer_options_key()), path))
+
+    ordered_paths = [path for _, path in sorted(trainer_options_path_tuples,
+        key=lambda trainer_options_path_pair: trainer_options_path_pair[0]['world_rank'])]
+
+    return ordered_paths
+
+def _add_or_update_sharded_key(state_key, state_value, state_sub_dict,
```

Comment:
if this is zero specific, add zero in the name to separate from all other types of helpers